Competition2017/src/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -0,0 +1,48 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToPositionCommand extends BaseCommand{
+
+    private final DriveSubsystem driveSubsystem;
+    private final PIDManager travelManager;
+    
+    private final DoubleProperty distanceTolerance;
+    private double targetPosition;
+    
+    @Inject
+    public DriveToPositionCommand(DriveSubsystem driveSubsystem, XPropertyManager propManager) {
+        this.driveSubsystem = driveSubsystem;
+        this.requires(driveSubsystem);
+        this.travelManager = new PIDManager("DriveToPosition", propManager, 0.1, 0, 0);
+        
+        distanceTolerance = propManager.createPersistentProperty("Distance Tolerance", 0.1);
+    }
+    
+    public void setTargetPosition(double targetPosition) {
+        this.targetPosition = driveSubsystem.convertToInches(targetPosition);
```

Comment:
These unit conversions scare me. It doesn't feel natural for the position to be in ticks here (I'd expect everything outside of the drive subsystem to be human units). If we do need to mix units, they should be documented.